ballerina/README.md

```diff
@@ -88,22 +88,22 @@ To use the Shopify connector in your Ballerina project, modify the `.bal` file a
 
 ### Step 1: Import the module
 
-Import the `ballerinax/shopify` module into your Ballerina project.
+Import the `ballerinax/shopify.admin` module into your Ballerina project.
 
 ```ballerina
-import ballerinax/shopify;
+import ballerinax/shopify.admin;
```

Comment:
For the examples, documentation, etc. shall we use an alias? 

```ballerina
import ballerinax/shopify.admin as shopadmin;
```

I am thinking out loud here, but this seems more specific than using just `admin`.

Reply:
Fixed in [58d5d58](https://github.com/ballerina-platform/module-ballerinax-shopify.admin/pull/8/commits/58d5d58a4eb004e23afdbef6bdb24c38be8d3bcb)